ci/testnet-sanity.sh

```diff
@@ -20,7 +20,10 @@ echo "--- $NET_URL: verify ledger"
 if [[ -z $NO_LEDGER_VERIFY ]]; then
   if [[ -d /var/snap/solana/current/config/ledger ]]; then
     # Note: here we assume this script is actually running on the leader node...
-    sudo solana.ledger-tool --ledger /var/snap/solana/current/config/ledger verify
+    LEDGER=/tmp/ledger-${BUILDKITE_BUILD_ID:-$$}
+    cp -r /var/snap/solana/current/config/ledger "$LEDGER"
+    solana.ledger-tool --ledger "$LEDGER" verify
```

Comment:
Plop the ledger copy into cwd so we don't need to worry about cleaning up /tmp when `solana.ledger-tool` returns non-zero?